tests/integration/replication.tcl

```diff
@@ -73,7 +73,7 @@ start_server {tags {"repl external:skip"}} {
         test {INCRBYFLOAT replication, should not remove expire} {
             r set test 1 EX 100
             r incrbyfloat test 0.1
-            after 1000
+            wait_for_ofs_sync $A $B
             assert_equal [$A debug digest] [$B debug digest]
```

Comment:
Isn't the purpose of the test case that it should be expired? Sync doesn't make it expire.

Reply:
i'm not sure what's the purpose of that test, but notice the expiry time is 100 seconds.

Reply:
yes, the expiry time is 100 seconds. it added in #6679, and wanted to fix #5256
i understand it want to make sure that the master and replica data are exactly the same

Reply:
Ah, sorry, I thought it's 100ms. Ignore this comment then!